websites/M/MarineTraffic/presence.ts

```diff
@@ -0,0 +1,330 @@
+const presence = new Presence({
+		clientId: "1013343014711738368",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let locationMap: string,
+	elemName: string,
+	shipStatus: string,
+	shipDest: string,
+	shipDep: string,
+	elemDesc: string;
+
+const capitalize = (s: string | never[]) =>
+	(s && s[0].toUpperCase() + s.slice(1)) || "";
+
+presence.on("UpdateData", async () => {
+	let presenceData: PresenceData = {
+		largeImageKey: "logo",
+		startTimestamp: browsingTimestamp,
+		details: "Viewing:",
```

Comment:
@EncryptedDev no **:** right?

Reply:
Since Encrypted just thumbsuped x')
```suggestion
		details: "Viewing",
```